tests/storage_pool/storage_pool_test.go

```diff
@@ -26,33 +26,71 @@ var _ = BeforeSuite(func() {
 	InitInstance()
 })
 
-var _ = Describe("{StoragePoolResize}", func() {
+var _ = Describe("{StoragePoolExpandDiskResize}", func() {
+
+	Context("When the author has both names", func() {
+		It("should get the existing pool", func() {
+			var poolID string
+			pools, err := Inst().V.ListStoragePools(metav1.LabelSelector{})
+			if err != nil {
+				Expect(err).NotTo(HaveOccurred())
+			}
+
+			Expect(len(pools)).NotTo(Equal(0))
+
+			for _, pool := range pools {
+				poolID = pool.Uuid
+				break
+			}
 
-	It("should get the existing pool", func() {
-		pools, err := Inst().V.ListStoragePools(metav1.LabelSelector{})
-		if err != nil {
+			poolToResize := pools[poolID]
+			oldSize := poolToResize.TotalSize
+			newSize := oldSize * 2 / units.GiB
+
+			err = Inst().V.ExpandPool(poolID, api.SdkStoragePool_RESIZE_TYPE_RESIZE_DISK, newSize)
+			Expect(err).NotTo(HaveOccurred())
+			time.Sleep(time.Second * 90)
```

Comment:
can we avoid sleeps and add wait until pool is resized? In some cases it could take more than 90 secods

Reply:
like add the backoff?

Reply:
not a backoff, but kind of ValidateStoragePool function which will expect the size of the pool and timeout if pool size will not be of that size.

Reply:
I have the same concept in storage pool validation for autoipilot